packages/prisma/migrations/20250618151833_add_uuid_to_app_routingforms_formresponse/migration.sql

```diff
@@ -0,0 +1,109 @@
+-- Step 1: Add column as nullable
+ALTER TABLE "App_RoutingForms_FormResponse" ADD COLUMN "uuid" TEXT;
+
+-- Step 2: Populate existing rows with UUIDs
+UPDATE "App_RoutingForms_FormResponse" SET "uuid" = gen_random_uuid()::text WHERE "uuid" IS NULL;
+
+-- Step 3: Make column NOT NULL
+ALTER TABLE "App_RoutingForms_FormResponse" ALTER COLUMN "uuid" SET NOT NULL;
+
+-- Step 4: Add unique constraint
+CREATE UNIQUE INDEX "App_RoutingForms_FormResponse_uuid_key" ON "App_RoutingForms_FormResponse"("uuid");
+
+-- Step 5: Add UUID to RoutingFormResponseDenormalized
+ALTER TABLE "RoutingFormResponseDenormalized" ADD COLUMN "uuid" TEXT;
+
+-- Step 6: Populate existing rows with UUIDs
+UPDATE "RoutingFormResponseDenormalized" d
+SET "uuid" = f."uuid"
+FROM "App_RoutingForms_FormResponse" f
+WHERE d."id" = f."id";
+
+-- Step 7: Make column NOT NULL
+ALTER TABLE "RoutingFormResponseDenormalized" ALTER COLUMN "uuid" SET NOT NULL;
+
+-- Step 8: Add unique constraint
+CREATE UNIQUE INDEX "RoutingFormResponseDenormalized_uuid_key" ON "RoutingFormResponseDenormalized"("uuid");
+
+-- Step 9: Add UUID to RoutingFormResponseDenormalized
```

Comment:
This part seems much, but actually only line 38 and 67 are added. The rest is the same as [packages/prisma/migrations/20250502113828_setup_triggers_for_routing_form_response_denormalized/migration.sql](https://github.com/calcom/cal.com/blob/3bea03f4b1fc6b612cc68cafb45d5c68ae971dd2/packages/prisma/migrations/20250502113828_setup_triggers_for_routing_form_response_denormalized/migration.sql#L17).